src/game-cards/index.ts

```diff
@@ -0,0 +1,99 @@
+import IGameCard from './game-card.interface';
+import { showCart, addToCart, checkCart } from '../utilites/cart';
+function gameCard({num, cover, name, publisher, price, year, platform, internet, genre, popular}: IGameCard) {
```

Comment:
try to use iGameCard: IGameCard instead (I don't think that you need to enumerate all properties)